spring-mvc-simple/pom.xml

```diff
@@ -115,6 +115,13 @@
             <groupId>org.json</groupId>
             <artifactId>json</artifactId>
             <version>${json.version}</version>
+        </dependency>
+		<dependency>
```

Comment:
@amit2103 why do we need this change?

Reply:
@pivovarit This PR is too old. We don't need it anymore.